core/src/main/java/org/apache/accumulo/fate/AdminUtil.java

```diff
@@ -390,11 +390,10 @@ private FateStatus getTransactionStatus(ReadOnlyTStore<T> zs, Set<Long> filterTx
 
       zs.unreserve(tid, 0);
 
-      if ((filterTxid != null && !filterTxid.contains(tid))
-          || (filterStatus != null && !filterStatus.contains(status)))
-        continue;
-
-      statuses.add(new TransactionStatus(tid, status, debug, hlocks, wlocks, top, timeCreated));
+      if ((filterTxid == null) || filterTxid.contains(tid) || (filterStatus == null)
+          || filterStatus.contains(status)) {
+        statuses.add(new TransactionStatus(tid, status, debug, hlocks, wlocks, top, timeCreated));
+      }
```

Comment:
For me, yes.  Can you verify by adding the accumulo.properties where you were testing / seeing the issue?

What I don't understand is why the filter seemed to change - needing the properties for a test using fate print is understandable, but the same filter code seems to be behavior seems to differ between the two branch and I don't see why.  That's why I set this to draft until the difference can be explained.

Reply:
It doesn't make sense to filter by both the txid AND the status. The txid should be unique. We should probably throw an error if the user provides both. Or if the user provides the txid, just filter by that.

Reply:
0 was a standing for any txid, could be 1234, whatever. The point was that specifying a set of txids and a state, then I would expect that it would be an AND - only the txids that match and only in the state provided.